### PR TITLE
add navigation polygon and precise view

### PR DESCRIPTION
**Pull request by lucienicolier**

### CODE REVIEW

File: documentation/how-to/navigation.en.md [hunk 1]
```diff
@@ -44,3 +47,10 @@ To activate this auto tracking feature, you can simply tap on the positioning bu
 !![](../assets/images/navigation-auto-tracking.png)
 
 This can be described as a simple *staking mode* functionality.
+
+## Positioning precise view
```
Comment (m-kuhn) [inline documentation/how-to/navigation.en.md:51]: Should we add "stakeout" to the title? This is what people usually look for

File: documentation/how-to/navigation.en.md [hunk 2]
```diff
@@ -44,3 +47,10 @@ To activate this auto tracking feature, you can simply tap on the positioning bu
...
+:material-tablet-android:{ .device-icon } Fieldwork
+
+Precise view (aka stakeout) panel which appears when a navigation to destination is active. appears when a navigation to destination is active.
```
Comment (m-kuhn) [inline documentation/how-to/navigation.en.md:54]: This line looks strange

File: documentation/how-to/navigation.en.md (hunk 1 above)
Comment (nirvn) [reply documentation/how-to/navigation.en.md:51]: +1 to add stakeout as a keyword in there, that's likely what many will search for to begin with.

**Comment by nirvn** [comment]
@lucienicolier , that's a speedy documentation process, thanks.

It might be worth expanding on the behavior of the precise view to mention the audio alarm in the form of a repeating ping with variable intervals across pings based on proximity (i.e. the jaws effect whereas the closer the shark gets to the human, the faster the ping repeats ;) ).

Also, when someone "hits the target", the precise view turns green. Hitting the target here is impacted by both the accuracy of the GNSS device as well as the precise view's selected precision. The logic is as follow: if the distance between the GNSS current position _minus the accuracy_ is less than 1/10th of the precise view's precision, we turn green.

Practical example: if the precise view is set to _1 meter_ and your GNSS accuracy is *perfect* (we all wish that was the case), the view would turn green when the position is less than 10cm from the destination.

**Comment by m-kuhn** [comment]
> Practical example: if the precise view is set to 1 meter and your GNSS accuracy is perfect (we all wish that was the case), the view would turn green when the position is less than 10cm from the destination.

Good idea to add an example. I would however choose one with real world relevance where the accuracy is not 0 (since that will never actually happen)

**Comment by lucienicolier** [comment]
@m-kuhn and @nirvn Thanks for the review. I'll improve it with your suggestion.

> @lucienicolier , that's a speedy documentation process, thanks.
> 
> It might be worth expanding on the behavior of the precise view to mention the audio alarm in the form of a repeating ping with variable intervals across pings based on proximity (i.e. the jaws effect whereas the closer the shark gets to the human, the faster the ping repeats ;) ).
> 
> Also, when someone "hits the target", the precise view turns green. Hitting the target here is impacted by both the accuracy of the GNSS device as well as the precise view's selected precision. The logic is as follow: if the distance between the GNSS current position _minus the accuracy_ is less than 1/10th of the precise view's precision, we turn green.
> 
> Practical example: if the precise view is set to _1 meter_ and your GNSS accuracy is _perfect_ (we all wish that was the case), the view would turn green when the position is less than 10cm from the destination.

Just tested, it is pretty impressive!

**Comment by nirvn** [comment]
@m-kuhn , @marioba , @lucienicolier , I've expanded the stakeout precise view section, I think this is ready for a final review and hopefully a merge soon after :)

**Comment by duke-nyuki** [comment]
Task linked: [QF-2028 Precision display for staking out](https://app.clickup.com/t/2192114/QF-2028)

File: documentation/how-to/navigation.en.md [hunk 3]
```diff
@@ -44,3 +47,16 @@ To activate this auto tracking feature, you can simply tap on the positioning bu
 !![](../assets/images/navigation-auto-tracking.png)
 
 This can be described as a simple *staking mode* functionality.
+
+## "Stakeout" precise view
+:material-tablet-android:{ .device-icon } Fieldwork
+
+QField's "stakeout" precise view panel appears when the current location to destination is less than a pre-defined threshold (by default 5 meters) and the positioning device has an accuracy that is less than half of that threshold. The precise view visiblity threshold value can be modified by long-pressing on the bottom-right navigation button and selecting the *Precise View Settings* menu item.
```
Comment (m-kuhn) [inline documentation/how-to/navigation.en.md:54]:
```suggestion
QField's "stakeout" precise view panel appears when the distance between the current location and the destination is less than a pre-defined threshold (by default 5 meters) and the positioning device has an accuracy that is less than half of that threshold. The precise view visibility threshold value can be modified by long-pressing on the bottom-right navigation button and selecting the *Precise View Settings* menu item.
```

Comment (nirvn) [reply documentation/how-to/navigation.en.md:54]: @m-kuhn , good changes, committed.

**Review by m-kuhn — APPROVED** [review]

**Pull request merged**